pkgs/kubeshark/kubeshark/registry.yaml

```diff
@@ -0,0 +1,21 @@
+packages:
+  - type: github_release
+    repo_owner: kubeshark
+    repo_name: kubeshark
+    asset: kubeshark_{{.OS}}_{{.Arch}}
+    description: The API traffic viewer for Kubernetes providing real-time, protocol-aware visibility into Kubernetes internal network capturing, dissecting and monitoring all traffic and payloads going in, out and across containers, pods, nodes and clusters.
+    supported_envs:
+      - windows/amd64
+      - darwin
+      - linux
+    overrides:
+      - goos: windows
+        goarch: amd64
+        asset: kubeshark.exe
+    checksum:
+      type: github_release
+      asset: kubeshark_{{.OS}}_{{.Arch}}.sha256
```

Comment:
You can use shorter `asset: "{{.Asset}}.sha256"` which is an alias to definition of `asset` in line 5